modules/images/src/lib/parsers/svg-utils.js

```diff
@@ -30,6 +24,13 @@ export function getBlobOrDataUrl(arrayBuffer, url) {
   return getBlob(arrayBuffer, url);
 }
 
-function isSVG(url) {
-  return url && (SVG_DATA_URL_PATTERN.test(url) || SVG_URL_PATTERN.test(url));
+export function getBlob(arrayBuffer, url) {
+  if (isSVG(url)) {
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=606319
+    // eslint-disable-next-line
+    console.error('SVG cannot be parsed directly to imagebitmap');
```

Comment:
Theoretically this branch should never be hit.  Throw instead?

Reply:
> Theoretically this branch should never be hit. Throw instead?

- Left this in for the eventuality that the Chromium bug was suddenly fixed. 
- But agree it can be confusing to the maintainer as this is not an active code path. 
- Updated to throw, as suggested.